packages/application-components/src/index.ts

```diff
@@ -24,6 +24,11 @@ export { default as TabularDetailPage } from './components/detail-pages/tabular-
 // Public page
 export { default as PublicPageLayout } from './components/public-page-layout';
 
+// Main Layout Page
+export { default as FormMainPage } from './components/main-pages/form-main-page';
+export { default as CustomFormMainPage } from './components/main-pages/custom-form-main-page';
+export { default as InfoMainPage } from './components/main-pages/info-main-page';
+
```

Comment:
Nit, can you move the `TabularMainPage` export in this group as well?

```suggestion
export { default as TabularMainPage } from './components/tabular-main-page';

```

Reply:
Updated [d3f1ce0](https://github.com/commercetools/merchant-center-application-kit/pull/2745/commits/d3f1ce099b3a568b5d0b212590aa009f89f89dc2)